google-cloud-spanner/docs/toc.json

```diff
@@ -32,26 +32,38 @@
     {
       "title": "Spanner",
       "type": "google/cloud/spanner",
-      "patterns": ["\/spanner", "\/spanner\/v1", "\/spanner\/admin\/instance\/v1", "\/spanner\/admin\/database\/v1", "google\/iam", "google\/protobuf", "google\/rpc"],
+      "patterns": ["\/spanner", "google\/iam", "google\/protobuf", "google\/rpc"],
       "nav": [
         {
+          "title": "Client",
+          "type": "google/cloud/spanner/client"
+        },
+        {
+          "title": "Results",
+          "type": "google/cloud/spanner/results"
+        },
+        {
+          "title": "Project",
+          "type": "google/cloud/spanner/project"
+        },
+        {
+          "title": "Instance",
+          "type": "google/cloud/spanner/instance"
+        },
+        {
+          "title": "Database",
+          "type": "google/cloud/spanner/database"
+        },
+        {
           "title": "V1",
           "type": "google/cloud/spanner/v1",
-          "patterns": ["\/spanner\/v1", "\/spanner\/admin\/instance\/v1", "\/spanner\/admin\/database\/v1", "google\/iam", "google\/protobuf", "google\/rpc"],
+          "patterns": ["\/spanner\/v1", "google\/iam", "google\/protobuf", "google\/rpc"],
           "nav": [
             {
               "title": "SpannerClient",
               "type": "google/cloud/spanner/v1/spannerclient"
             },
             {
-              "title": "InstanceAdminClient",
```

Comment:
Why were these removed? Spanner has 3 APIs, I think it would be good to show all 3.